adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala

```diff
@@ -159,17 +162,39 @@ case class GenotypeRDD(rdd: RDD[Genotype],
   /**
    * Explicitly saves to VCF.
    *
+   * @param args Arguments defining where to save the file.
+   * @param stringency The validation stringency to use when writing the VCF.
+   *   Defaults to LENIENT.
+   */
+  def saveAsVcf(args: ADAMSaveAnyArgs,
```

Comment:
Hmm, yeah I do see that.

So you're suggesting no format guessing `save` methods for `GenotypeRDD`, `VariantRDD`, and `VariantContextRDD`.  `GenotypeRDD` and `VariantRDD` have only `saveAsParquet` and `VariantContextRDD` has only `saveAsVcf`.  The format guessing happens in the CLI classes.  I could support that.